scripts/run_nodes.sh

```diff
@@ -101,7 +101,7 @@ run_node() {
     --validator-port ${validator_port} \
     -laleph-party=debug \
     -laleph-network=debug \
-    -lvalidator-network=debug \
+    -lclique-network=debug \
```

Comment:
Would be great to have it automatized by all `LOG_TARGET` s

Reply:
That would require quite a bit of `sed`/`awk` magic, I think it's a good idea, but probably not for this PR.

Maybe we could get an ADR for logging targets and then executing it would also include magicking up the appropriate script here?